__tests__/index.test.ts

```diff
@@ -56,7 +56,7 @@ describe('Run', () => {
   });
 });
 
-describe('Run - cli args', () => {
+describe.skip('Run - cli args', () => {
```

Comment:
These tests tie up our run API with CLI which is incorrect, I will clean this up or will do a follow up PR to get rid of this and add proper tests for CLI for arguments.

Reply:
Fixed now.